src/components/EvidenceRequestTable.tsx

```diff
@@ -7,14 +7,16 @@ export const EvidenceRequestTable: FunctionComponent<Props> = ({
 }) => {
   const rows = useMemo(
     () =>
-      requests.map((row) => {
-        return {
-          id: row.id,
-          resident: row.resident.name,
-          document: row.documentTypes[0].title,
-          made: `${row.createdAt.toRelative()}`,
-        };
-      }),
+      requests
+        .sort((a, b) => (a.createdAt < b.createdAt ? 1 : -1))
```

Comment:
As we use the same function in `ResidentsTable` would it be possible to extract it to a function. That would avoid duplication, and we could test it to ensure it sorts records as we expect